tools/depends/configure.ac

```diff
@@ -73,13 +73,14 @@ AC_ARG_WITH([sdk-path],
 
 AC_ARG_WITH([sdk],
   [AS_HELP_STRING([--with-sdk],
-  [specify sdk platform version (optional for android). default is android-24])],
+  [specify sdk platform version.])],
   [use_sdk=$withval])
 
 AC_ARG_WITH([ndk-api],
   [AS_HELP_STRING([--with-ndk-api],
-  [specify ndk level (optional for android). default is 21])],
-  [use_ndk_api=$withval])
+  [specify ndk level (optional for android), default is 21.])],
+  [use_ndk_api=$withval],
+  [use_ndk_api=21])
```

Comment:
@Rechi Normal?

Reply:
just moved from a few lines below

Reply:
Mmm.. The line below says
if use_ndk_api.empty()
  use_ndk_api = 21

no?
That's different than hardcoding it to 21.

Reply:
```
${parameter:-word}

    If parameter is unset or null, the expansion of word is substituted. Otherwise, the value of parameter is substituted.

```

Reply:
Oh, nvm.
Didn't saw it was in the macro. Default value, I guess...